src/utils/config.ts

```diff
@@ -1,2 +1,4 @@
 export const getApiUrl = (subdomain: string, zone: string) =>
 	`https://${subdomain}.${zone}.qasphere.com`
+
+export const REQUIRED_NODE_VERSION = '20.15.0' // lts/iron
```

Comment:
How much earlier can we go? even in my machine node is 20.10

Usually lots of QA have outdated versions of runtimes, because they do not do any software development professionally and typically have a life. 

I am not saying we have to support node 0.4 (there is a scandal on the internet about a shady guy who tries to add node 0.4 support to svelte packages). 

But maybe we can support node 16 or 18 without much changes ?

Reply:
https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json

Isn't this is a more standard approach to do this

Reply:
We can go as early as node v18 without breaking or changing anything however:

Node v18: `File` class is throws an error (can be substituted with Blob); shows an experimental warning for using `FormData`.
Node v16: throws an error since `fetch` API is introduced in v17.5

We can however implement or use some packages to polyfill, if we decide to support older versions.

For now I can put `v20.x.x`  as supported node.js version?

Reply:
We should try to support atleast v18.
https://endoflife.date/nodejs
It will be supported till Apr 2025 in maintenance mode.

Reply:
@satvik007 Currently everything works on v18. But it will show warnings when using `--attachments` flag because of `FormData` class.

I could probably install some npm package to polyfill formdata, but I'm not sure if it's necessary since it's not really breaking anything.

What do you suggest I do?

Reply:
@ramilamparo @satvik007 Warnings from Node 18 are tolerable. If things are working, let's support node 18 then. We can always extend support to older versions if there would be any requests. 

The worst offender here is Ubuntu: 22.04 installs node 12 by default. Ubuntu is going to be supported until 2023, probably some poor guy will back port security fixes to that old node the whole time.